ktor-server/ktor-server-jetty/build.gradle

```diff
@@ -15,6 +15,7 @@ kotlin {
             api project(':ktor-server:ktor-server-core')
             api project(':ktor-server:ktor-server-test-host')
             api project(':ktor-server:ktor-server-test-suites')
+            api project(':ktor-server:ktor-server-tomcat')
```

Comment:
Ooops, why do we depend on tomcat in jetty?

Reply:
Simply because I did not know, what's the right module for the created multi engine test :)

Reply:
Fixed. I created a abstract class in server-testing-suites and in each engine testSrc a subclass